src/scss/06_components/pages/_basic-page.scss

```diff
@@ -182,19 +182,70 @@ $tag-gap: $spacing-half;
   margin-top: 0;
 
   // stylelint-disable-next-line no-descending-specificity
-  a {
+  a,
```

Comment:
Now that tags is growing into it's own building block, maybe we should move them out from basic page and into `06_components/_tags.scss`, what do you think ?